ee/fe/modules/500-openvpn/templates/openvpn/configmap.yaml

```diff
@@ -29,7 +32,45 @@ data:
     topology subnet
     proto tcp
     port 1194
-    dev tun0
+    dev tun-tcp
```

Comment:
Maybe, `d8-ovpn-tcp`?

Reply:
There will be no collisions, we do not use hostNetwork=true.